proprietary/tokens/src/components/ams/file-input.tokens.json

```diff
@@ -0,0 +1,32 @@
+{
+  "ams": {
+    "file-input": {
+      "color": { "value": "{ams.color.primary-black}" },
+      "font-family": { "value": "{ams.text.font-family}" },
+      "font-size": { "value": "{ams.text.level.5.font-size}" },
+      "line-height": { "value": "{ams.text.level.5.line-height}" },
+      "outline-offset": { "value": "{ams.focus.outline-offset}" },
+      "placeholder": {
+        "color": { "value": "{ams.color.neutral-grey3}" }
+      },
+      "disabled": {
+        "color": { "value": "{ams.color.neutral-grey2}" }
+      },
+      "file-selector-button": {
+        "background-color": { "value": "{ams.color.primary-white}" },
+        "color": { "value": "{ams.color.primary-blue}" },
+        "box-shadow": { "value": "inset 0 0 0 {ams.border.width.md} {ams.color.primary-blue}" },
```

Comment:
`cursor` is used in the CSS, but it's not defined here. Although it's probably a better idea to define the cursor for the whole input

Reply:
Very iffy solution, defining a cursor on the input causes it to flicker, the button will not inherit (not seen as child of input) any CSS.